releasewarrior/wiki_data.py

```diff
@@ -303,6 +307,13 @@ def update_inflight_issue(data, resolve, logger):
     return data
 
 
+def update_inflight_graphid(data, phase, graphid, logger):
+    data = deepcopy(data)
+    current_build_index = get_current_build_index(data)
+    data["inflight"][current_build_index]["graphids"].append([phase, graphid])
```

Comment:
Should we add ability to overwrite a phase graphid instead of always appending? I'm thinking of cases where we put the wrong graphid in or need a new one. This can be fixed manually via sync for now I think.

Reply:
Sure. #63